src/frontend/js/pages/DashboardContracts/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect } from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { Pagination, usePagination } from 'components/Pagination';
+import { Spinner } from 'components/Spinner';
+import Banner, { BannerType } from 'components/Banner';
+import { DashboardItemContract } from 'widgets/Dashboard/components/DashboardItem/Contract';
+import { useContracts } from 'hooks/useContacts';
+
+const messages = defineMessages({
+  loading: {
+    defaultMessage: 'Loading contracts...',
+    description: 'Message displayed while loading contracts',
+    id: 'components.DashboardContracts.loading',
+  },
+  empty: {
+    defaultMessage: 'You have no contract yet.',
+    description: 'Message displayed when there are no contracts',
+    id: 'components.DashboardContracts.empty',
+  },
+});
+
+export const DashboardContracts = () => {
```

Comment:
It looks odd to me to have a `Dashboard...` component outside of `widgets/Dashboard`, nope?

Reply:
it's a page, it's on `/page`